jenkins/tenant/aws/ovn/run_policies.py

```diff
@@ -0,0 +1,105 @@
+import os
+from ast import literal_eval
+
+QUAY_CLOUD_GOVERNANCE_REPOSITORY = os.environ.get('QUAY_CLOUD_GOVERNANCE_REPOSITORY',
+                                                  'quay.io/cloud-governance/cloud-governance')
+
+
+# available_policies: Run policies in dry_run="yes" mode
+
+
+def run_cmd(cmd: str):
+    """
+    This method run the command
+    :param cmd:
+    :type cmd:
+    :return:
+    :rtype:
+    """
+    os.system(cmd)
+
+
+access_key = os.environ['access_key']
+secret_key = os.environ['secret_key']
+s3_bucket = os.environ.get('s3_bucket')
+account_name = os.environ['account_name']
+days_to_delete_resource = os.environ.get('days_to_delete_resource', 14)
+LDAP_HOST_NAME = os.environ['LDAP_HOST_NAME']
+LOGS = os.environ.get('LOGS', 'logs')
+ALERT_DRY_RUN = os.environ.get('ALERT_DRY_RUN', False)
+ES_HOST = os.environ['ES_HOST']
+ES_PORT = os.environ['ES_PORT']
+ADMIN_MAIL_LIST = os.environ.get('ADMIN_MAIL_LIST', '')
+
+# Set es_index if given
+ES_INDEX = os.environ.get('ES_INDEX', None)
+env_es_index = f'-e es_index={ES_INDEX}' if ES_INDEX else ''
+
+
+def get_container_cmd(env_dict: dict):
+    create_container_envs = lambda item: f'-e {item[0]}="{item[1]}"'
+    env_list = ' '.join(list(map(create_container_envs, env_dict.items())))
+    container_name = "cloud-governance-poc-haim"
+    container_run_cmd = f"""
+podman run --rm --name "{container_name}" --net="host" {env_list} {env_es_index} {QUAY_CLOUD_GOVERNANCE_REPOSITORY}
+"""
+    return container_run_cmd
+
+
+policies_in_action = ['zombie_cluster_resource',
+                      'ip_unattached',
+                      # 'zombie_snapshots',
+                      'unused_nat_gateway',
+                      # 'empty_roles',
+                      'unattached_volume']
+
```

Comment:
@athiruma, only 4 action policies ?

Reply:
@ebattat: Yes, 4 action policies for now.